google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableCloudMonitoringExporter.java

```diff
@@ -216,19 +220,12 @@ private CompletableResultCode exportBigtableResourceMetrics(Collection<MetricDat
     }
 
     ProjectName projectName = ProjectName.of(bigtableProjectId);
-    CreateTimeSeriesRequest bigtableRequest =
-        CreateTimeSeriesRequest.newBuilder()
-            .setName(projectName.toString())
-            .addAllTimeSeries(bigtableTimeSeries)
-            .build();
-
-    ApiFuture<Empty> future =
-        this.client.createServiceTimeSeriesCallable().futureCall(bigtableRequest);
+    ApiFuture<List<Empty>> future = exportTimeSeries(projectName.toString(), bigtableTimeSeries);
```

Comment:
Since you dont care about the actual returned value I think you want `ApiFuture<?>`